x-pack/legacy/plugins/lens/public/app_plugin/app.tsx

```diff
@@ -87,6 +105,11 @@ export function App({
 
   const lastKnownDocRef = useRef<Document | undefined>(undefined);
 
+  // Sync Kibana breadcrumbs any time we save the document
```

Comment:
nit: comment should be `// Sync Kibana breadcrumbs any time we change the title of the saved document`

Reply:
True.